Bottom: 4677128c654542650d457a25456e4d4ec953d7d4
Top:    b1f1bba1e2ecfc7229e861b5f03ae29daf71cbf6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-16 23:51:41 +0000

Adjust STEP export to 0.2mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 6bc2257..ec12573 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -44,7 +44,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 
 
 HID step_hid;
